app/views/course/discussion/topics/_topics.html.slim

```diff
@@ -0,0 +1,4 @@
+- @topics.each do |topic|
+  = render partial: topic.actable_type.demodulize.underscore, locals: { topic: topic }
```

Comment:
I have a render collection that accepts a prefix. Use that and then render the collection. We do this for lesson plan also.